src/haptic_feedback.c

```diff
@@ -145,9 +148,16 @@ void haptic_feedback_update(
         float speed = VESC_IF->mc_get_speed();
         const CfgHapticTone *tone = get_haptic_tone(hf);
         if (tone->strength > 0.0f) {
-            VESC_IF->foc_play_tone(
-                0, tone->frequency, scale_strength(tone->strength, hf->cfg, speed)
-            );
+            float strength = scale_strength(tone->strength, hf->cfg, speed);
+            if (type_to_play == HAPTIC_FEEDBACK_ERROR_HI_VOLTAGE) {
+                float duration = current_time - hf->start_time;
+                if (duration > 3) {
+                    // more than 3 seconds in HV? Start scaling up for next 2 seconds
+                    // up to 50% louder
+                    strength = strength * fminf(1.5, duration * 0.25 + 0.25);
+                }
+            }
+            VESC_IF->foc_play_tone(0, tone->frequency, strength);
```

Comment:
This is a second special-case being added :smiling_face_with_tear:.

Also, this raises the strength above the user-set threshold, and by as much as 50%. Doesn't seem like a good thing to do.

If I was to contrive a disaster scenario, someone configures their haptic feedback and tests it at the configured levels (no way to easily know what this +50% strength will do and even that it can actually happen on HV). Then, months later, they trigger HV on a steep downhill. They don't react in time, the loud haptics mess up the imu and make the nose go higher, they go into taildrag and crash.